integration_files/SweRV_EH1/Makefile

```diff
@@ -54,7 +54,7 @@ ITERATIONS          := 0
 # LSF CMD
 LSF_CMD             :=
 # Generator timeout limit in seconds
-TIMEOUT             := 1800
+TIMEOUT             := 10000
```

Comment:
@amnafayyaz-lm Isn't it a too big value?

Reply:
For now keeping timeout 1k until further advancements in pyflow. Here is the timeout point discussed with google's RiscV dv : [https://github.com/google/riscv-dv/issues/783](url)